### ISSUE TITLE
panic in secrets-bridge-server logs when creating pods with label - secrets.bridge.enabled: "true" but without "secrets.bridge.k8s.path" label.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#5983]
Rancher Version:v1.1.4-rc1

Set up secrets-bridge-server and secret-bridge-agent by following instruction in https://github.com/rancher/secrets-bridge/blob/master/docs/setup.md  for k8s environment.

Create pods with label - secrets.bridge.enabled: "true" but without "secrets.bridge.k8s.path" label.

I see the following panic in secrets-bridge-server logs:

```
9/15/2016 4:03:54 PMtime="2016-09-15T23:03:54Z" level=info msg="Verifing: 05e3b197-a31b-486f-9e43-013462189edc"
9/15/2016 4:03:54 PM2016/09/15 23:03:54 http: panic serving 10.42.157.92:59448: interface conversion: interface is nil, not string
9/15/2016 4:03:54 PMgoroutine 38 [running]:
9/15/2016 4:03:54 PMnet/http.(*conn).serve.func1(0xc82000c200)
9/15/2016 4:03:54 PM    /usr/local/go/src/net/http/server.go:1389 +0xc1
9/15/2016 4:03:54 PMpanic(0xa05680, 0xc82043aa80)
9/15/2016 4:03:54 PM    /usr/local/go/src/runtime/panic.go:443 +0x4e9
9/15/2016 4:03:54 PMgithub.com/rancher/secrets-bridge/verifier.(*RancherK8sVerifiedResponse).PrepareResponse(0xc82000e870, 0xc82000e701, 0xc82016a000, 0xc82007fb00, 0x0, 0x0)
9/15/2016 4:03:54 PM    /go/src/github.com/rancher/secrets-bridge/verifier/k8sResponse.go:27 +0x36c
9/15/2016 4:03:54 PMgithub.com/rancher/secrets-bridge/verifier.(*RancherVerifier).Verify(0xc820024000, 0xc82000e780, 0x0, 0x0, 0x0, 0x0)
9/15/2016 4:03:54 PM    /go/src/github.com/rancher/secrets-bridge/verifier/client.go:77 +0x5d1
9/15/2016 4:03:54 PMgithub.com/rancher/secrets-bridge/bridge.ContainerStart(0x7fe3c8483a48, 0xc8204cc000, 0xc82000e780, 0x0, 0x0)
9/15/2016 4:03:54 PM    /go/src/github.com/rancher/secrets-bridge/bridge/server.go:147 +0x7f
9/15/2016 4:03:54 PMgithub.com/rancher/secrets-bridge/bridge.messageHandler(0x7fe3c8483a48, 0xc8204cc000, 0xc8200ba000)
9/15/2016 4:03:54 PM    /go/src/github.com/rancher/secrets-bridge/bridge/server.go:120 +0x40c
9/15/2016 4:03:54 PMgithub.com/rancher/secrets-bridge/bridge.tokenVerificationHandler(0x7fe3c8483a48, 0xc8204cc000, 0xc8200ba000)
9/15/2016 4:03:54 PM    /go/src/github.com/rancher/secrets-bridge/bridge/server.go:91 +0x15e
9/15/2016 4:03:54 PMnet/http.HandlerFunc.ServeHTTP(0xc12188, 0x7fe3c8483a48, 0xc8204cc000, 0xc8200ba000)
9/15/2016 4:03:54 PM    /usr/local/go/src/net/http/server.go:1618 +0x3a
9/15/2016 4:03:54 PMgithub.com/rancher/secrets-bridge/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc82000e820, 0x7fe3c8483a48, 0xc8204cc000, 0xc8200ba000)
9/15/2016 4:03:54 PM    /go/src/github.com/rancher/secrets-bridge/vendor/github.com/gorilla/mux/mux.go:107 +0x285
9/15/2016 4:03:54 PMnet/http.serverHandler.ServeHTTP(0xc82000c980, 0x7fe3c8483a48, 0xc8204cc000, 0xc8200ba000)
9/15/2016 4:03:54 PM    /usr/local/go/src/net/http/server.go:2081 +0x19e
9/15/2016 4:03:54 PMnet/http.(*conn).serve(0xc82000c200)
9/15/2016 4:03:54 PM    /usr/local/go/src/net/http/server.go:1472 +0xf2e
9/15/2016 4:03:54 PMcreated by net/http.(*Server).Serve
9/15/2016 4:03:54 PM    /usr/local/go/src/net/http/server.go:2137 +0x44e
```

**Comment by cloudnautique** [comment]
Should be resolved in rancher/secrets-bridge:v0.1.2 in the latest catalog entry.

**Comment by sangeethah** [comment]
Tested with rancher/secrets-bridge:v0.1.1
This issue is not seen anymore.
